Codo-Baggins/covid_comparisons
# Testing/initial tests

## What's this PR do?
- Creates render tests for `App`, `USAState`, `Stat`, `ComparisonContainer`
- Add format date method to `App` (this was necessary for `USAState` testing)

## Where should the reviewer start?
- `App.test`, `USAState.test`, `Stat.test`, `ComparisonContainer.test`, `formatDate` in `App`

## How should this be manually tested?
- run test suite, all tests should pass
- in browser, on page load, date should be formatted '01/01/2021'
